src/vite-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="vite/client" />
+
+declare module '*.svg' {
```

Comment:
i think you don't need this declaration (and the scss one too), `/// <reference types="vite/client" />` should take care of both